### PR TITLE
Projectile Refactoring 6

### PR DESCRIPTION
**Pull request by MrRowey**
File Tidy-ups & File Refactoring

### CODE REVIEW

File: projectiles/ShellTankTerran01/ShellTankTerran01_script.lua [hunk 1]
```diff
@@ -1,8 +1,6 @@
---
 -- script for projectile TankShell
---
-local Projectile = import("/lua/sim/projectile.lua").Projectile
-ShellTankTerran01 = Class(Projectile) {
+local TankProjectile = import("/lua/sim/projectile.lua").Projectile
```
Comment (Garanas) [inline projectiles/ShellTankTerran01/ShellTankTerran01_script.lua:2]: Why was this renamed from `Projectile`?

Comment (MrRowey) [reply projectiles/ShellTankTerran01/ShellTankTerran01_script.lua]: not sure why either xD, reverted :)

**Review by Garanas — APPROVED** [review]
